game.rb

```diff
@@ -0,0 +1,91 @@
+class Game
+
+board={A1: " ",B1: " ",C1: " ",A2: " ",B2: " ",C2: " ",A3: " ",B3: " ",C3: " "}
+
+def self.print_board(board)
+  puts " | A | B | C |"
+  puts " -------------"
+  puts "1| #{board[:A1]} | #{board[:B1]} | #{board[:C1]} |"
+  puts " -------------"
+  puts "2| #{board[:A2]} | #{board[:B2]} | #{board[:C2]} |"
+  puts " -------------"
+  puts "3| #{board[:A3]} | #{board[:B3]} | #{board[:C3]} |"
+  puts " -------------"
+end
+
+def self.consistent_rows?(board)
+  row1 = board[:A1] == board[:B1] && board[:B1] == board[:C1] && board[:A1] != " "
+  row2 = board[:A2] == board[:B2] && board[:B2] == board[:C2] && board[:A2] != " "
+  row3 = board[:A3] == board[:B3] && board[:B3] == board[:C3] && board[:A3] != " "
+  row1||row2||row3
+end
+
+def self.consistent_columns?(board)
+  column1 = board[:A1]==board[:A2]&&board[:A2]==board[:A3] && board[:A1] != " "
+  column2 = board[:B1]==board[:B2]&&board[:B2]==board[:B3] && board[:B1] != " "
+  column3 = board[:C1]==board[:C2]&&board[:C2]==board[:C3] && board[:C1] != " "
+  column1||column2||column3
+end
+
+def self.consistent_diagonals?(board)
+  diagonal1 = board[:A1]==board[:B2]&&board[:B2]==board[:C3] && board[:A1] != " "
+  diagonal2 = board[:C1]==board[:B2]&&board[:B2]==board[:A3] && board[:C1] != " "
+  diagonal1||diagonal2
+end
+
+def self.victory?(board)
+  consistent_rows?(board)||consistent_columns?(board)||consistent_diagonals?(board)
+end
+
+def self.move_x(board)
+  input=gets.chomp.upcase
```

Comment:
te metody nie powinny pobierać inputu od użytkownika - powinny dostawać jako parametr input. to uprościłoby znacząco testy